CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this crate are documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this crate adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.6.1]
```

Comment:
Should we add the date?

Reply:
Yes, I'll do that before I'll merge it (so that the date will actually correspond to the day it'll be released :P)